src/hooks/use-prompt-usage.tsx

```diff
@@ -42,7 +41,7 @@ const calculateDaysRemaining = ( quotaResetDate: string ): number => {
 export function PromptUsageProvider( { children }: PromptUsageProps ) {
 	const { Provider } = promptUsageContext;
 
-	const [ promptLimit, setPromptLimit ] = useState( LIMIT_OF_PROMPTS_PER_USER );
+	const [ promptLimit, setPromptLimit ] = useState( 0 );
```

Comment:
This change is modifying the case when the user is offline and opens the settings modal.
Instead of the actual limit (200) it appears 0.

From my point of view, it's better to keep the previous number (200) and display the correct number.

It's tru that the promptCount is 0, so we can consider displaying another message when the user is offline.

| Before | After |
|--------|--------|
|  <img width="1012" alt="Screenshot 2024-10-30 at 19 56 48" src="https://github.com/user-attachments/assets/46d71b01-3119-4a9d-be5e-911205ff638d"> | <img width="1012" alt="Screenshot 2024-10-30 at 19 56 13" src="https://github.com/user-attachments/assets/99f1d41a-ec7f-4dd6-addb-885a36e44afc"> |


```suggestion
	const [ promptLimit, setPromptLimit ] = useState( LIMIT_OF_PROMPTS_PER_USER );
```

Reply:
Oh, I hadn't considered the offline case; Will remember it for future cases. We should definitely cover such situations.

> It's tru that the promptCount is 0, so we can consider displaying another message when the user is offline.

I noticed that mostly we are using just simple message `You're currently offline.` and I think it's good option for this place, to show that Assistant is unavailable. WDYT?

Changes are made. Check it out please.